README.md

```diff
@@ -35,27 +36,27 @@ optional arguments:
                         Path to a directory where the generated content is
 ```
 
-### using `nox`
+## Using Nox
 
-This project uses `nox` as a task runner to run the code generator. You can install `nox` and run `build_lsp` session to generate code from spec available in the repo.
+This project uses Nox as a task runner to run the code generator. You can install Nox and run a `build_lsp` session to generate code from the spec available in this repo.
 
 ```console
 > python -m pip install nox
 > nox --session build_lsp
 ```
 
-You can format code, run tests, and other tasks using `nox` as well.
+You can also use Nox to format code, run tests and run various tasks. Run `nox --list` to see all available tasks.
 
 # Contributing plugins
 
 ## Adding a new plugin
 
-Follow these steps to generate boiler plate code for new plugin:
+Follow these steps to generate a boilerplate code for a new plugin:
 
-1. Create a virtual environment for python using python 3.8 and activate that environment.
-    1. If you have python extension for VS Code installed then run `Python: Create Environment` command. Be sure to select all the `requirements.txt` files in the repo. This should, install all packages needed and select the environment for you.
+1. Create a virtual environment for Python using Python >= 3.8 and activate that environment.
+    1. If you are using the Python extension for VS Code, you can just run the **Python: Create Environment** command from the Command Palette. Be sure to select all the `requirements.txt` files in the repo. This command will install all packages needed and select the newly created environment for you.
 1. Ensure `nox` is installed.
-    1. Run `nox --list`, is nox is installed oyu should see a list of available sessions. Otherwise, run `python -m pip install nox` from the python 3.8 environment you created above.
+    1. Run `nox --list` in the terminal. If Nox is installed you should see a list of all available sessions. Otherwise, run `python -m pip install nox` in the activated environment you created above.
```

Comment:
```suggestion
    1. Run `nox --list` in the terminal. If Nox is installed, you should see a list of all available sessions. Otherwise, run `python -m pip install nox` in the activated environment you created above.
```